clickhouse/network.xml

```diff
@@ -0,0 +1,4 @@
+<clickhouse>
+    <listen_host>0.0.0.0</listen_host>
+    <keep_alive_timeout>10</keep_alive_timeout>
```

Comment:
Increases the keep-alive timeout from 3 (default) to 10 seconds. This would prevent closed connections on big responses.